zk-token-sdk/src/instruction/transfer_with_fee.rs

```diff
@@ -363,6 +380,10 @@ impl Verifiable for TransferWithFeeData {
     }
 }
 
+impl ZkProofContext for TransferWithFeeProofContext {
+    const LEN: usize = 650;
```

Comment:
Yep 😅...

Thinking about it now, we can perhaps store the hash of the proof context rather than the entire bytes of the context. The application program can reconstruct the context state that it is expecting, hash it, and then compare it to the stored context.

Reply:
Sorry, this is about as far as my knowledge goes... how would that work? I thought you need access to private data (ie keys and balances) in order to generate the proofs.

Reply:
After internal discussion, hashing the context state was not worth the extra complications for the value it provides.